pkg/ccl/importccl/import_stmt.go

```diff
@@ -504,16 +504,18 @@ func importPlanHook(
 			}
 
 			// Validate target columns.
-			var intoCols []string
-			var isTargetCol = make(map[string]bool)
+			var intoCols []uint32
+
+			var isTargetCol = make(map[sqlbase.ColumnID]bool)
 			for _, name := range importStmt.IntoCols {
 				var err error
-				if _, err = found.FindActiveColumnByName(name.String()); err != nil {
-					return errors.Wrap(err, "verifying target columns")
+				active, err := found.FindActiveColumnsByNames(tree.NameList{name})
+				if err != nil {
+					return errors.Wrapf(err, "verifying target columns: norm=%q str=%q", name.Normalize(), name.String())
 				}
 
-				isTargetCol[name.String()] = true
```

Comment:
could we just change `name.String()` to `active.Name` rather than switch everything to IDs right now?

Reply:
I tried doing that... The entire pipeline was getting stuck anyway.  We would need to re-resolve the names because we serialize those names into the proto.

Reply:
right, we serialize the names, but my suggestion is we should be serializing `columnDesc.Name`, which is exactly the same string we'll compare to on the other end, right?